services/s3/src/test/java/software/amazon/awssdk/services/s3/functionaltests/GetBucketPolicyFunctionalTest.java

```diff
@@ -27,15 +27,16 @@
 import org.junit.Test;
 import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
 import software.amazon.awssdk.auth.credentials.StaticCredentialsProvider;
+import software.amazon.awssdk.crt.Log;
 import software.amazon.awssdk.regions.Region;
 import software.amazon.awssdk.services.s3.S3AsyncClient;
 import software.amazon.awssdk.services.s3.S3AsyncClientBuilder;
 import software.amazon.awssdk.services.s3.S3Client;
 import software.amazon.awssdk.services.s3.S3ClientBuilder;
+import software.amazon.awssdk.services.s3.S3CrtAsyncClientBuilder;
 import software.amazon.awssdk.services.s3.model.GetBucketPolicyResponse;
 
 public class GetBucketPolicyFunctionalTest {
-    private static final URI HTTP_LOCALHOST_URI = URI.create("http://localhost:8080/");
```

Comment:
Do you have the bandwidth to convert these two classes to JUnit5? It'd be helpful for our migration. [BlobUnmarshallingTest.java](https://github.com/aws/aws-sdk-java-v2/blob/master/test/codegen-generated-classes-test/src/test/java/software/amazon/awssdk/services/protocolrestjson/BlobUnmarshallingTest.java) is an example with both parameterized tests and Wiremock.

Reply:
Unfortunately no, I agree and I'd normally do it for new tests. Do you think this is a blocker?

Reply:
No, that's fine.